src/Core/BaseKernel.php

```diff
@@ -243,6 +245,17 @@ protected function bootErrorHandling()
         }
     }
 
+    /**
+     * Register a subscriber for the symfony/mailer MailerEvent
+     * https://symfony.com/doc/current/components/event_dispatcher.html#using-event-subscribers
+     */
+    protected function bootEmail()
+    {
+        $subscriber = MailerSubscriber::create();
+        $dipatcher = Injector::inst()->get(EventDispatcherInterface::class . '.mailer');
+        $dipatcher->addSubscriber($subscriber);
+    }
```

Comment:
Why not do this in yaml?

If this is done here, should it be more generic like `bootEventSubscribers()`, or more specific like `bootEmailSubscriber()`?

Reply:
Have moved to yml